cloud/misc/proxmox_kvm.py

```diff
@@ -0,0 +1,1055 @@
+    choices: ['lsi', 'lsi53c810', 'virtio-scsi-pci', 'virtio-scsi-single', 'megasas', 'pvscsi']
+    required: false
+    default: null
+    type: string
+  serial:
+    description:
+      - A hash/dictionary of serial device to create inside the VM. C('{"key":"value", "key":"value"}').
+      - Keys allowed are - serial[n](str; required) where 0 ≤ n ≤ 3.
+      - Values allowed are - C((/dev/.+|socket)).
+      - /!\ If you pass through a host serial device, it is no longer possible to migrate such machines - use with special care.
+    default: null
+    required: false
+    type: A hash/dictionary defining serial
+  shares:
+    description:
+      - Rets amount of memory shares for auto-ballooning. (0 - 50000).
+      - The larger the number is, the more memory this VM gets.
+      - The number is relative to weights of all other running VMs.
+      - Using 0 disables auto-ballooning, this means no limit.
+    required: false
+    default: null
+    type: integer
+  skiplock:
+    description:
+      - Ignore locks
+      - Only root is allowed to use this option.
+    required: false
+    default: null
+    choices: [ "yes", "no" ]
+    type: boolean
+  smbios:
+    description:
+      - Specifies SMBIOS type 1 fields.
+    required: false
+    default: null
+    type: string
+  sockets:
+    description:
+      - Sets the number of CPU sockets. (1 - N).
+    required: false
+    default: 1
+    type: integer
+  startdate:
+    description:
+      - Sets the initial date of the real time clock.
+      - Valid format for date are C('now') or C('2016-09-25T16:01:21') or C('2016-09-25').
+    required: false
+    default: null
+    type: string
+  startup:
+    description:
+      - Startup and shutdown behavior. C([[order=]\d+] [,up=\d+] [,down=\d+]).
+      - Order is a non-negative number defining the general startup order.
+      - Shutdown in done with reverse ordering.
+    required: false
+    default: null
+    type: string
+  state:
+    description:
+      - Indicates desired state of the instance.
+      - If C(current), the current state of the VM will be fecthed. You can acces it with C(results.status)
+    choices: ['present', 'started', 'absent', 'stopped', 'restarted','current']
+    required: false
+    default: present
+  tablet:
+    description:
+      - Enables/disables the USB tablet device.
+    required: false
+    choices: [ "yes", "no" ]
+    default: "no"
+    type: boolean
+  tdf:
+    description:
+      - Enables/disables time drift fix.
+    required: false
+    default: null
+    choices: [ "yes", "no" ]
+    type: boolean
+  template:
+    description:
+      - Enables/disables the template.
+    required: false
+    default: "no"
+    choices: [ "yes", "no" ]
+    type: boolean
+  timeout:
+    description:
+      - Timeout for operations.
+    default: 30
+    required: false
+    type: integer
+  validate_certs:
+    description:
+      - Enables/disables HTTPS certificate verification for API access.
```

Comment:
Can you please change this to the following so it's consistent with other modules and the security implication is clearer:
"If C(no), SSL certificates will not be validated. This should only be used on personally controlled sites using self-signed certificates."